lib/action_logic/action_validation/presence_validation.rb

```diff
@@ -6,13 +6,17 @@ module ActionValidation
     class PresenceValidation < BaseValidation
 
       def self.validate!(validation_rules, context)
-        return unless validation_rules.values.find { |expected_validation| expected_validation[:presence] }
-        errors = presence_errors(validation_rules, context)
-        raise ActionLogic::PresenceError.new(errors) if errors.any?
+        tmp_rules = validation_rules.clone
```

Comment:
we have to `clone` a copy of `validation_rules` here instead of `deleting` the `:raise_action_logic_exception` directly, since for 'around' validations, `validation_rules` will be accessed twice, once before and once after.